src/main/java/com/odysseusinc/athena/model/athena/DownloadShareId.java

```diff
@@ -0,0 +1,54 @@
+package com.odysseusinc.athena.model.athena;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Embeddable
```

Comment:
Why composite key was used? What is the advantages of solution?

Reply:
fixed